test/cql/cdc_enable_disable.cql

```diff
@@ -0,0 +1,23 @@
+-- create cdc enabled table
+create table ks.t (pk int, ck int, v int, primary key(pk, ck)) with cdc = {'enabled': true};
+
+-- this write goes to the log
+insert into ks.t (pk, ck, v) values (1, 1, 100);
+
+-- disable cdc. should retain the log
+alter table ks.t with cdc = {'enabled': false};
+
+-- add more data to base - should not generate log
+insert into ks.t (pk, ck, v) values (2, 2, 200);
+
+-- should still work, and give one row (1, 1, 100)
+select "cdc$batch_seq_no", "cdc$operation", "cdc$ttl", pk, ck, v from ks.t_scylla_cdc_log;
+
+-- turn cdc back on
+alter table ks.t with cdc = {'enabled': true};
```

Comment:
I would add one more step before turning CDC back on: adding another column or removing existing column (best do both). See if the log table adjusts after turning CDC back on (I checked locally and it does seem to adjust, but it's better to have a regression test).